### ISSUE TITLE
undefined method `size' for nil:NilClass with version 6.9.0.363 

### ISSUE THREAD

**Issue by zog** [newrelic/rpm#323]
Since we migrated to the latest (6.9.0.363 ), we sometimes have the following error:
```
NoMethodError: undefined method `size' for nil:NilClass
In new_relic/rack/browser_monitoring.rb in traced_call at line 
```
The corresponding line:

```
headers[CONTENT_LENGTH] = response_string.size.to_s
```

**Comment by tannalynn** [comment]
Hi @zog 
Just wanted to let you know that we will be including your [pull request(https://github.com/newrelic/rpm/pull/324) in the next release of the agent. 
We will let you know when that version is released!
Thanks!

**Comment by rachelleahklein** [comment]
The fix for this issue has been included in version 6.10.0 of the Ruby agent, released today! Thanks again for bringing it to our attention!
